File: Android_Study/Example/DoitMission-09_EnterInformation/app/src/main/java/com/satelite54/doit_09/MainActivity.java
package com.satelite54.doit_09;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Fragment1 fragment1 = new Fragment1();
        getSupportFragmentManager().beginTransaction().replace(R.id.container, fragment1).commit();
    }
}

File: Android_Busan_it/MyToast/app/src/main/java/com/example/mytoast/MainActivity.kt
package com.example.mytoast

import android.content.Context
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Gravity
import android.view.WindowManager
import android.widget.Button
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    lateinit var btn1 : Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var tMsg = Toast.makeText(this, "토스트 연습", Toast.LENGTH_SHORT)

        var display = (getSystemService(Context.WINDOW_SERVICE) as WindowManager).defaultDisplay
        var xOffset = (Math.random() * display.width).toInt()
        var yOffset = (Math.random() * display.height).toInt()

        tMsg.setGravity(Gravity.TOP or Gravity.LEFT , xOffset, yOffset)

        btn1 = findViewById<Button>(R.id.btn1)

        btn1.setOnClickListener {
            tMsg.show()
        }
    }
}